src/engine/UIManager.cxx

```diff
@@ -288,30 +282,19 @@ void UIManager::drawUI() const
 
 void UIManager::toggleGroupVisibility(const std::string &groupID, UIElement *sender)
 {
-  if (groupID.empty())
-  {
-    LOG(LOG_ERROR) << "UI Element with Action \"ToggleVisibilityOfGroup\" was called without a parameter.";
-    return;
-  }
-
   if (sender)
   {
     Button *button = dynamic_cast<Button *>(sender);
+
     // cast the object to a Button to check if it's a toggle button.
-    if (button && button->getUiElementData().isToggleButton)
-    {
-      for (const auto &it : m_uiGroups[groupID])
-      {
+    if(button && button->getUiElementData().isToggleButton)
+      for(const auto &it : m_uiGroups[groupID])
         it->setVisibility(button->checkState());
-      }
-      return;
-    }
+    return;
```

Comment:
This was the problem @adct-the-experimenter

Reply:
Thanks! I tested it and settings is working now.